lib/plugins/aws/package/compile/events/apiGateway/lib/hack/updateStage.js

```diff
@@ -216,6 +223,17 @@ function handleTracing() {
   this.apiGatewayStagePatchOperations.push(operation);
 }
 
+function handleMetrics() {
+  if (!this.hasMetricsConfigured) return;
+  const metricsEnabled = this.state.service.provider.apiGateway.metrics;
+
+  let operation = { op: 'replace', path: '/*/*/metrics/enabled', value: 'false' };
+  if (metricsEnabled) {
+    operation = { op: 'replace', path: '/*/*/metrics/enabled', value: 'true' };
+  }
```

Comment:
It'll be nicer to make it one liner as:

```javascript
operation = { op: 'replace', path: '/*/*/metrics/enabled', value: metricsEnabled ? 'true' : 'false' };
```

Reply:
@medikoo 
Thank you for your suggestion. You're right.
I have just changed the code like below,

https://github.com/serverless/serverless/pull/7754/files#diff-b8e198657dea88d759b31561660e36c4R230-R234

The reason why the code I have changed is 5 lines even if your request code is 1 line is prettier automatically changed it.

Please check it out when you have time.